tools/perf/chrome_releases.json

```diff
@@ -0,0 +1,25 @@
+  "100.0.4896.30": {"url": "https://dl.google.com/release2/chrome/adk24dr3625psdygaxigvgxcsniq_100.0.4896.30/100.0.4896.30_chrome_installer.exe"},
+  "100.0.4896.46": {"url": "https://dl.google.com/release2/chrome/d6t7uc36wtapqylvl3x2ekglsq_100.0.4896.46/100.0.4896.46_chrome_installer.exe"},
+  "100.0.4896.56": {"url": "https://dl.google.com/release2/chrome/h3lzewmzpbsnis7wqww2zcki3y_100.0.4896.56/100.0.4896.56_chrome_installer.exe"},
+  "100.0.4896.60": {"url": "https://dl.google.com/release2/chrome/adk7ykd4m3hxmsbdd7g6j5pjn7gq_100.0.4896.60/100.0.4896.60_chrome_installer.exe"},
+  "101.0.4951.15": {"url": "http://dl.google.com/release2/chrome/e2k5pta4jpg7l7btgpbqbdylm4_101.0.4951.15/101.0.4951.15_chrome_installer.exe"},
+  "101.0.4951.26": {"url": "http://dl.google.com/release2/chrome/acccw4bs5ssird7xryhfelciqi_101.0.4951.26/101.0.4951.26_chrome_installer.exe"},
+  "101.0.4951.34": {"url": "https://dl.google.com/release2/chrome/oalppmwx4lb3hw3kpgfbrlwr2u_101.0.4951.34/101.0.4951.34_chrome_installer.exe"},
+  "101.0.4951.41": {"url": "https://dl.google.com/release2/chrome/adhsa4gplkf573nizk5shcj4dsfq_101.0.4951.41/101.0.4951.41_chrome_installer.exe"},
+  "101.0.4951.54": {"url": "https://dl.google.com/release2/chrome/m4ugl2oo7k3e3c7k3m4c6g43uu_101.0.4951.54/101.0.4951.54_chrome_installer.exe"},
+  "102.0.5005.40": {"url": "https://dl.google.com/release2/chrome/acxbxxlc22rou5m54kvm5yavpxfq_102.0.5005.40/102.0.5005.40_chrome_installer.exe"},
+  "102.0.5005.49": {"url": "https://dl.google.com/release2/chrome/iezafe4rwyfc433thdfz4bqpou_102.0.5005.49/102.0.5005.49_chrome_installer.exe"},
+  "102.0.5005.61": {"url": "https://dl.google.com/release2/chrome/advox7uyndj3xruhbfinh6syiy5q_102.0.5005.61/102.0.5005.61_chrome_installer.exe"},
+  "103.0.5060.24": {"url": "https://dl.google.com/release2/chrome/ac45c6gdhxpntfwaegpfnzqsyhga_103.0.5060.24/103.0.5060.24_chrome_installer.exe"},
+  "103.0.5060.53": {"url": "https://dl.google.com/release2/chrome/adpllftgiudg2qsog7pdou2vggiq_103.0.5060.53/103.0.5060.53_chrome_installer.exe"},
+  "103.0.5060.114": {"url": "https://dl.google.com/release2/chrome/adtma6pi6kul22k4nxj3ds6hikgq_103.0.5060.114/103.0.5060.114_chrome_installer.exe"},
+  "103.0.5060.134": {"url": "https://dl.google.com/release2/chrome/adysojpcmdixd4hvz3fcla42tuja_103.0.5060.134/103.0.5060.134_chrome_installer.exe"},
+  "104.0.5112.29": {"url": "https://dl.google.com/release2/chrome/a2inljvpirdkwwsvcf4k2uqxsu_104.0.5112.29/104.0.5112.29_chrome_installer.exe"},
+  "104.0.5112.57": {"url": "https://dl.google.com/release2/chrome/adco26zns6ti5tjgpmxy4r47c5da_104.0.5112.57/104.0.5112.57_chrome_installer.exe"},
+  "104.0.5112.81": {"url": "https://dl.google.com/release2/chrome/l63wbcwf2balpdrthyxxt3c6hy_104.0.5112.81/104.0.5112.81_chrome_installer.exe"},
+  "105.0.5195.28": {"url": "https://dl.google.com/release2/chrome/acptoylgmvl5tcm27wvl7rwieifq_105.0.5195.28/105.0.5195.28_chrome_installer.exe"}
```

Comment:
is it assumed to be updated manually?

Reply:
Yep, from https://get-chrome.herokuapp.com/channel/beta
Currently we need update it only few times per chrome major release. Not sure that we what to establish a CI job to do it, because now there is a gap between chrome beta and brave nightly => better to build our chromium builds.

I believe we could make a simple script to update the file later.

Reply:
imo the manual process is fine, just need to document the whole thing somewhere